application/controllers/tags_controller.py

```diff
@@ -13,6 +13,32 @@ class TagsController(ApplicationController):
 
     method_decorators = [check_es_connection]
 
+    @use_args(tags_validator.multi_tagging_args())
+    def multi_tagging(self, params):
+        """ Add tags to multiple artifacts """
+
+        new_multi_tags = list(params["tags"])
+        existing_multi_tags = []
+        artifacts = []
+
+        for object_id in set(params["ids"]):
```

Comment:
Please use Artifact.find_all instead.
Elasticsearch uses mget for multiple id's